scripts/test-kic-2.x-upgrade.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+# ------------------------------------------------------------------------------
+# test-kick-2.x-upgrade.sh
+#
+# This script is temporary: in the timeframe between 1.x and 2.x KIC
+# releases this script was made to validate that upgrading to the
+# latest pre-release image was functional, once 2.0 releases fully
+# this script will be removed because all tests thereafter will be
+# running on that version and it's lineage.
+# ------------------------------------------------------------------------------
+
+# ------------------------------------------------------------------------------
+# Environment Variables
+# ------------------------------------------------------------------------------
+
+if [[ -z $TEST_ENV_NAME ]]
+then
+    TEST_ENV_NAME="kong-charts-tests"
+fi
+
+# ------------------------------------------------------------------------------
+# Shell Configuration
+# ------------------------------------------------------------------------------
+
+set -euo pipefail
+
+# ------------------------------------------------------------------------------
+# Kubernetes Cluster Setup
+# ------------------------------------------------------------------------------
+
+kubectl cluster-info --context kind-${TEST_ENV_NAME} 1>/dev/null
+KUBERNETES_VERSION="$(kubectl version -o json | jq -r '.serverVersion.gitVersion')"
```

Comment:
nit: use `-o jsonpath=...`

Reply:
unfortunately the `version` subcommand doesn't support the same inspection flags as the `get` subcommand does, or yes I would have added this (or in my case, `-o gotemplate=='<TEMPLATE'` of which I am most comfortable with).

Reply:
Ah, my bad.